app/javascript/mastodon/components/status_list.js

```diff
@@ -41,16 +42,26 @@ class StatusList extends ImmutablePureComponent {
     } else if (this.props.onScroll) {
       this.props.onScroll();
     }
-  }
+  }, 200, {
+    trailing: true,
+  });
```

Comment:
This prevents TIMELINE_SCROLL_TOP and loading more processing until they stops scroll. For example, below (trivial?) issues will happens:

- TIMELINE_SCROLL_TOP: The unread indicator won't appears when new toot received while scrolling.
- loading more: Fetch request will be delayed about 200ms.